src/hydra/fetchResource.js

```diff
@@ -0,0 +1,13 @@
+import fetchJsonLd from "./fetchJsonLd";
+import get from "lodash.get";
+
+export default async entrypointUrl => {
+  return await fetchJsonLd(entrypointUrl, { items_per_page: 0 }).then(
```

Comment:
By default, the name of this attribute in API Platform core is `itemsPerPage`, we should use it here.

Reply:
Done